docs/datastores.md

```diff
@@ -0,0 +1,57 @@
+Datastores
+==========
+
+Version 1.0+ supports using Postgres as a storage backend instead of the
+deprecated NeDB storage backend. 
+
+NeDB End-of-life
+--------
+
+NeDB is a library which is used to store json documents locally to disk to give the bridge some local persistent state. 
+All deployments of this bridge before `1.0` will have been using NeDB.
+
+Starting with version `1.0`, NeDB will be deprecated and shouldn't be used for new installations. NeDB is
+[unmaintained](https://github.com/matrix-org/matrix-appservice-bridge/issues/77) and doesn't scale well for the
+needs of this bridge. Features such as puppeting will not be supported, however existing functionality will continue
+to be maintained until support for NeDB is removed. 
+
+Version 1.0 of the bridge only supports postgres as an alternative datastore.
+
+Using PostgreSQL
+----------------
+
+You must first create a fresh database on an postgresql instance, create a user and grant the user
+permission on the bridge:
+
+```sql
+CREATE DATABASE slack_bridge;
+CREATE USER slackbridge_user WITH PASSWORD 'something very secret';
+GRANT ALL PRIVILEGES ON DATABASE "slack" to slackbridge_user;
+```
+
+You should then update the config with details about the new database.
+
+```yaml
+db:
+   engine: "postgres"
+   connectionString: "postgresql://slackbridge_user:pass@localhost/slack_bridge?sslmode=require"
+```
+
+(replacing pass with the password set above)
```

Comment:
Can we use the same string for "something very secret" and "pass" - makes it clearer they correspond